File: 5KYU/Moving Zeros To The End/solution.cs
using System.Linq;
public class Kata
{
	public static int[] MoveZeroes(int[] arr)
	{
		var result = arr.Where(i => i != 0).ToList();
		result.AddRange(Enumerable.Repeat(0, arr.Length - result.Count));
		return result.ToArray();
	}
}

File: 5KYU/Moving Zeros To The End/solution.go
package kata

func MoveZeros(arr []int) []int {
	res := make([]int, len(arr))
	ci := 0
	zerosCount := 0
	for i := 0; i < len(arr); i++ {
		if arr[i] == 0 {
			zerosCount = zerosCount + 1
		} else {
			res[ci] = arr[i]
			ci = ci + 1
		}
	}

	for i := 0; i < zerosCount; i++ {
		res[ci] = 0
		ci = ci + 1
	}

	return res
}
